collections/_article/parallax-progress-report.md

```diff
@@ -0,0 +1,120 @@
+
+#### The right time to scroll to me
+
+`autoscroll` scrolls the layer *automatically* in addition to the parallax settings. Honestly, that's it! This could be done with a custom script, but it was highly requested, easy to implement, and had no performance hit if you weren't using it. So, why not?
+
+<video autoplay loop muted playsinline title="An example of parallax scrolling with autoscroll enabled">
+  <source src="/storage/blog/parallax2d/autoscroll.mp4" type="video/mp4">
+</video>
+
+#### Positive growth
+
+This shouldn't come up often, but the new `repeat_times` property is being introduced to help with the problem of zooming out while trying to achieve an infinite repeat effect. When you zoom out and the repeat size becomes smaller than the screen, the effect is broken! 
+
+<video autoplay loop muted playsinline title="An demonstration of the parallax effect zoomed out">
+  <source src="/storage/blog/parallax2d/zoom_out.mp4" type="video/mp4">
+</video>
+
+Increasing the `repeat_times` will allow for the number of repeats to grow and spread outward, so that you can zoom out and not reveal the magic going on behind the scenes. Eventually, it'd be nice to have a flag to have this adjust automatically, but this is a good start!
+
+#### What else?
+
+`Parallax2D` is now compatible with `BackBufferCopy` and `CanvasGroup`, and you're also free to rotate the camera without breaking the parallax effect! Some things needed shifted around in the `ParallaxBackground` code to accommodate the new changes and I discovered a nice surprise. Through some basic tests with 20 complex layers (comparing before and after the feature was added), I saw a good performance boost with both the old system and the new one with the changes... which is nice!
+
+### Caveats and Future
+
+With any effect that is based on perception, there are some tricky areas (since the entire effect is a trick!). A few of the same drawbacks to `ParallaxBackground` still apply to `Parallax2D`. For example, multiple cameras aren't directly supported (boooo!). This makes sense because `Parallax2D` follows the same concept as `ParallaxBackground`: Textures move at different speeds relative to the viewport. If you have multiple viewports, the textures can't move relative to all of them and can't be in two places at once! Just like in `ParallaxBackground`, as a workaround, you can clone the `Parallax2D`s and place each in a separate viewport. Besides the other future improvements mentioned earlier in this article, I'm also looking for ideas to better visualize parallax in the editor, whether that comes in the form of guides or another editor tool.
```

Comment:
```suggestion
With any effect that is based on perception, there are some tricky areas (since the entire effect is a trick!). A few of the same drawbacks to `ParallaxBackground` still apply to `Parallax2D`. For example, multiple cameras aren't directly supported (boooo!). This makes sense because `Parallax2D` follows the same concept as `ParallaxBackground`: textures move at different speeds relative to the viewport. If you have multiple viewports, the textures can't move relative to all of them and can't be in two places at once! Just like in `ParallaxBackground`, as a workaround, you can clone the `Parallax2D`s and place each in a separate viewport. Besides the other future improvements mentioned earlier in this article, I'm also looking for ideas to better visualize parallax in the editor, whether that comes in the form of guides or another editor tool.
```